airgap/server/server.go

```diff
@@ -161,3 +162,22 @@ func (b *airGapServerImpl) ObtainMetadata(ctx context.Context, options *airgap.T
 
 	return &txMetadata, nil
 }
+
+func (b *airGapServerImpl) FilterQuery(ctx context.Context, options *airgap.FilterQueryParams) ([]types.Log, error) {
+	var addresses []common.Address
+	for _, contract := range options.Contracts {
+		address, err := b.srvCtx.addressFor(ctx, registry.ContractID(contract), options.FromBlock)
+		if err != nil {
+			return nil, fmt.Errorf("'Contract' not a valid registry ID")
+		}
+		addresses = append(addresses, address)
+	}
+
+	query := ethereum.FilterQuery{
+		Addresses: addresses,
+		FromBlock: options.FromBlock,
+		ToBlock:   options.ToBlock,
+		Topics:    options.Topics,
```

Comment:
I imagine it would be nice for the `topic` event signatures to be specified by event name and similarly `addresses` by celo contract names
I was under the impression the coinbase consumers of these interfaces did not want to be encoding using our contract ABIs

Reply:
the addresses also *might* change during the provided range due to a governance action on the registry and this could lead to misleading response data